Sources/SwiftInfoCore/SlackFormatter.swift

```diff
@@ -3,13 +3,18 @@ import Foundation
 public struct SlackFormatter {
     public init() {}
 
-    public func format(output: Output, projectInfo: ProjectInfo) -> (json: [String: Any], message: String) {
-        let json = output.summaries.map { $0.slackDictionary }
-        let prefix = "SwiftInfo results for \(projectInfo.description):"
+    public func format(output: Output, titlePrefix: String?, projectInfo: ProjectInfo) -> (json: [String: Any], message: String) {
         let errors = "\nErrors:\n\(output.errors.joined(separator: "\n"))"
-        let title = prefix + (output.errors.isEmpty ? "" : errors)
+
+        let title1 = titlePrefix.flatMap { $0 + "\n" } ?? ""
+        let title2 = "SwiftInfo results for \(projectInfo.description):"
+        let title3 = output.errors.isEmpty ? "" : errors
```

Comment:
We can go further and make `SwiftInfo results for` replaceable too, what do you think? I thought that's what you meant by prefix here.

The "new" default message could be `"SwiftInfo results (\(projectInfo.description))`, with your property replacing the `SwiftInfo results ` portion, like this:

`public func format(output: Output, titlePrefix: String = "SwiftInfo results for "`

Reply:
(Or the default parameter could be in sendToSlack, whatever you think fits best)

Reply:
I made the entire prefix replaceable and rebased.

One thing that might require more explanation is why I avoided using a default value in the method parameter, as that would require logic in the less-easily tested SwiftInfo.swift callsite to see whether the parameter value is nil or not. I also tried putting the default prefix in a default value for the method parameter in sendToSlack, but then that value has to be shared with other sites calling SlackFormatter by using a static property for example. I found this to be awkward and complex, choosing to use a simple optional property with a default value in the _body_ of `format` instead of in any method parameters.

https://github.com/rockbruno/SwiftInfo/pull/60/files#diff-124a8e1a75b0a09e1e7fe2a37981311a2c4a60ee281259b0521562926d742626R6